packages/@ama-sdk/schematics/schematics/ng-update/typescript/v10.3/update-openapiversion.ts

```diff
@@ -0,0 +1,75 @@
+import type {Rule} from '@angular-devkit/schematics';
+import * as semver from 'semver';
+import {getWorkspaceConfig, globInTree} from '@o3r/schematics';
+import {chain} from '@angular-devkit/schematics';
+
+/**
+ * Update open api version used in the project to match with the one used in @ama-sdk/schematics:typescript-core
+ */
+export const updateOpenApiVersionInProject = (): Rule => {
```

Comment:
The schematic seems a bit tricky and based on complex regexp, maybe it would help to provide UT for it :S